platform_versioned_docs/version-23.4.0/functionality_matrix/functionality_matrix.mdx

```diff
@@ -5,26 +5,29 @@ date: "21 Apr 2023"
 tags: [compatibility, nextflow, nf-launcher]
 ---
 
-Each Seqera Platform version makes use of `nf-launcher` to determine the Nextflow version used as its baseline. This Nextflow version can be overridden with the `NXF_VER` environment variable in your `nextflow.config` file, but note that Seqera may not work reliably with Nextflow versions other than the baseline version.
+Each Seqera Platform version makes use of `nf-launcher` to determine the Nextflow version used as its baseline. You can override this version with the `NXF_VER` environment variable during pipeline launch, but note that Seqera may not work reliably with Nextflow versions other than the baseline version.
 
-We officially support the two latest major releases (22.3.x, 22.4.x, etc) at any given time.
-
-nf-launcher versions prefixed with j17 refer to Java version 17; j11 refers to Java 11.
+The two most recent major versions (23.4.x, 23.3.x, etc.) of Seqera Platform are supported at any given time.
 
 | Platform version | nf-launcher version | Nextflow version | Fusion version |
 | ---------------- | ------------------- | ---------------- | -------------- |
+| *                |                     | 24.04.2          | 2.3            |
 | 23.4.4           | j17-23.10.1         | 23.10.1          | 2.2            |
 | 23.4.3           | j17-23.10.1         | 23.10.1          | 2.2            |
 | 23.4.2           | j17-23.10.1         | 23.10.1          | 2.2            |
 | 23.4.1           | j17-23.10.1         | 23.10.1          | 2.2            |
 | 23.3.0           | j17-23.10.0         | 23.10.0          | 2.1            |
 | 23.2.0           | j17.23.04.2-up3     | 23.04.2          | 2.1            |
 | 23.1.3           | j17-23.04.1         | 23.04.1          | 2.1            |
-| 22.4.1           | j17-22.10.6         | 22.10.6          | -              |
-| 22.4.0           | j17-22.10.6         | 22.10.6          | -              |
+
+:::note 
+Nextflow edge versions after 24.04.2 use Fusion 2.4 by default. 
+:::
 
 ---
 
+To use a Nextflow version other than the baseline in your pipeline run, use a [pre-run script](./launch/advanced#pre-and-post-run-scripts) during launch. For example, enter `export NXF_VER=24.04.2` in the **Pre-run script** field to take advantage of [new Nextflow features](https://nextflow.io/blog/2024/nextflow-2404-highlights.html) before a Platform version is released with 24.04.2 as its baseline. 
```

Comment:
You've got NXF_VER stuff above the table too. Does it make sense to have this split across the page?

Reply:
Maybe keep the NXF_VER stuff light here (i..e just mention you can overrride) and then link to the `launch/advanced.mdx` page for actual details how to do it.

Reply:
Simplified.